modules/microshift-adding-service-to-blueprint.adoc

```diff
@@ -9,16 +9,25 @@
 
 Adding the {microshift-short} RPM package to an Image Builder blueprint enables the build of a {op-system-ostree} image with {microshift-short} embedded.
 
+* Follow step 1 to create your own minimal blueprint file which results in a faster minimal {microshift-short} installation.
+* Follow Step 2 to use the generated blueprint for installation which includes all the RPM packages and container images. This is a longer installation process, but a faster start up because container references are accessed locally.
```

Comment:
```suggestion
* Follow step 2 to use the generated blueprint for installation which includes all the RPM packages and container images. This is a longer installation process, but a faster start up because container references are accessed locally.
```